test/testHelpers.js

```diff
@@ -19,10 +19,10 @@ import HexUtils from '../src/common/components/hexgrid/HexUtils';
 
 import { attackBotCard } from './data/cards';
 
-export function getDefaultState() {
+export function getDefaultState(gameMode = null) {
   const state = cloneDeep(defaultGameState);
-  const deck = [instantiateCard(attackBotCard)].concat(collection);
-  const simulatedGameStartAction = {type: socketActions.GAME_START, payload: {decks: {orange: deck, blue: deck}}};
+  const deck = [instantiateCard(attackBotCard)].concat(collection.slice(1, DECK_SIZE));
+  const simulatedGameStartAction = {type: socketActions.GAME_START, payload: {decks: {orange: deck, blue: deck}, gameMode}};
```

Comment:
I think there might be enough nested objects here that it'd be better formatted to spread it out over a few lines

Reply:
Fix'd